deps/amqp10_client/src/amqp10_client_connection.erl

```diff
@@ -287,6 +287,22 @@ open_sent({call, From}, begin_session,
           #state{pending_session_reqs = PendingSessionReqs} = State) ->
     State1 = State#state{pending_session_reqs = [From | PendingSessionReqs]},
     {keep_state, State1};
+open_sent(_EvtType, {close, Reason}, State) ->
+    %% TODO: stop all sessions writing
```

Comment:
This comment is wrong because no sessions have begun at this point.

Reply:
I pushed a commit which removes this comment.

Reply:
Thanks!